app/templates/components/frost-bunsen-input-select.hbs

```diff
@@ -0,0 +1,21 @@
+<div class={{labelWrapperClassName}}>
+  <label class="alias">{{renderLabel}}</label>
+  {{#if required}}
+    <div class='required'>Required</div>
+  {{/if}}
+</div>
+<div class={{inputWrapperClassName}}>
+  {{frost-select
+    disabled=cellConfig.disabled
+    onInput=(action "onChange")
+    onChange=(action "onChange")
+    data=options
+    selectedValue=value
+  }}
+</div>
+{{#if errorMessage}}
+  <div class={{labelWrapperClassName}}></div>
+  <div class="error">
+    {{errorMessage}}
+  </div>
+{{/if}}
```

Comment:
I believe this should actually be
```handlebars
{{#if errorMessage}}
  <div>
    <div class={{labelWrapperClassName}}></div>
      <div class="error">
        {{errorMessage}}
     </div>
  </div>
{{/if}}
```
otherwise I believe the error message will be under the label instead of the input.